tests/tests.js

```diff
@@ -12,81 +26,81 @@ CD._searchString = function() {
   return 'foo=bar&baz%20test=quux%20value';
 };
 
-var imageSuccessStub = function() {
-  var self = this;
+const imageSuccessStub = function() {
+  const self = this;
   setTimeout(function() {
```

Comment:
Couldn't you just use an ES6 arrow function instead of defining `self`?

Reply:
Unfortunately not, because these two functions are declared at the top of the test file, absent of any scope. There's no clear way to capture the scope inside the imageSuccessStub besides grabbing the context at the moment. We could do it with a closure, but that looks messier than just capturing the `this` value.

if these functions were defined on an actual object or in a class, arrow functions would work